### DISCUSSION TITLE
aci-exporter 0.7.0

### DISCUSSION THREAD

**Discussion by thenodon** [opsdis/aci-exporter#52] [General] [answered]
With release 0.7.0 it is now possible to configure queries in multiple files in a specific configuration directory, default `config.d`. I  have tried to collect different queries, including the quereis @camrossi  used in his cisco devnet presentation. I have tried to put them in different files by category. I hope that everybody can contribute with their favorite queries and also additional  categories. I is of course possible to have quereis in the config.yaml file and it even possible to generate a single config.yaml from the config.d using the `-default` argument to `aci-exporter`. 
From 0.7.0 aci-exporter is created as a container image based on the repo Dockerfile, thanks to @minefuto. The container is stored at Githubs container registry, ghcr.io. To get the latest just run: 
```shell
docker pull ghcr.io/opsdis/aci-exporter:latest
```
Or select the version using the version tag, like `aci-exporter:v0.7.0`

**Comment by thenodon** [comment]
If you have some great dashboards you like to share it would be great. I will refactor the ones in `grafana_dashboards` and include the work the @camrossi put into the `examples` directory so they are easy to import into Grafana. If you add dashboards it would be super if we all try to use the same template variables so its easy to create navigation links. These are the ones I have used:
```
$fabric - is the fabric name used in the configuration file. It is part of all timeseries
$podid - the pod id
$nodeid - the node id 
```
If you have som cool dashboard that you like to share, discuss etc you can share it here https://github.com/opsdis/aci-exporter/discussions/53

**Comment by tomicer-git** [comment]
Hi Anders. I did some testing on the tool because there is a need to implement similar solution and it is really good. Thanx for great efforts. I have one question - does this get some updates or how is it maintained ? For example if I would raise an issue (for example, something not working or similar), would this be even checked or modified etc. Best regards, Tomi

**Reply by thenodon** [reply, marked as answer]
Hi @tomicer-git. The reason that there has been very little activity on this project is that there has not been any bug or feature request for a long time. Hopefully that means that it works. So if you find any bugs I will try to fix it, but in the spirit of opensource you are also welcome to create PR. But its important to understand this is opensource and not a commercial support deal. 
Hope that answer your question. And if you like to bring something to aci-exporter community please share your use case, any benefits the project gave compared to other solutions etc. Sharing is a two way process.
Regards 
Anders
